packages/web3-eth-contract/src/contract.ts

```diff
@@ -211,69 +210,9 @@ export class Contract<Abi extends ContractAbi>
 	public readonly options: ContractOptions;
 
 	/**
-	 * Can be used to set {@link Contract.defaultAccount} for all contracts.
+	 * Set to true if you want contracts' defaults to sync with global defaults.
 	 */
-	public static defaultAccount?: HexString;
-
-	/**
-	 * Can be used to set {@link Contract.defaultBlock} for all contracts.
-	 */
-	public static defaultBlock?: BlockNumberOrTag;
-
-	/**
-	 * Can be used to set {@link Contract.defaultHardfork} for all contracts.
-	 */
-	public static defaultHardfork?: string;
-
-	/**
-	 * Can be used to set {@link Contract.defaultCommon} for all contracts.
-	 */
-	public static defaultCommon?: Common;
-
-	/**
-	 * Can be used to set {@link Contract.transactionSendTimeout} for all contracts.
-	 */
-	public static transactionSendTimeout?: number;
-
-	/**
-	 * Can be used to set {@link Contract.transactionBlockTimeout} for all contracts.
-	 */
-	public static transactionBlockTimeout?: number;
-
-	/**
-	 * Can be used to set {@link Contract.transactionConfirmationBlocks} for all contracts.
-	 */
-	public static transactionConfirmationBlocks?: number;
-
-	/**
-	 * Can be used to set {@link Contract.transactionPollingInterval} for all contracts.
-	 */
-	public static transactionPollingInterval?: number;
-
-	/**
-	 * Can be used to set {@link Contract.transactionPollingTimeout} for all contracts.
-	 */
-	public static transactionPollingTimeout?: number;
-
-	/**
-	 * Can be used to set {@link Contract.transactionReceiptPollingInterval} for all contracts.
-	 */
-	public static transactionReceiptPollingInterval?: number;
-
-	/**
-	 * Can be used to set {@link Contract.transactionConfirmationPollingInterval} for all contracts.
-	 */
-	public static transactionConfirmationPollingInterval?: number;
-
-	/**
-	 * Can be used to set {@link Contract.blockHeaderTimeout} for all contracts.
-	 */
-	public static blockHeaderTimeout?: number;
-
-	/**
-	 * Can be used to set {@link Contract.handleRevert} for all contracts.
-	 */
-	public static handleRevert?: boolean;
+	public static sync_with_globals = false;
```

Comment:
What's the reasoning for this being a `static` property? I believe it would make more sense for it to be a public property for each instance with an option to set the initial value by setting `ContractInitOptions.syncWithContext` upon instantiation

Additionally, we don't use snake casing for properties, only constants

Reply:
Being in `ContractInitOptions` isn't sufficient?  Why should we have it in two places( init options and public value)?